Preference Access SSN Matching/LETTERSPECS/PREFACCESS.SSNMATCHING.CFG

```diff
@@ -0,0 +1,155 @@
+*******************************************************************************
+* PREFACCESS.SSNMATCHING.CFG
+*
+* This letterfile contains parameters for the RB.PREFACCESS.SSNMATCHING
+* specfile. Each line starting with an asterisk will be considered a comment
+* so feel free to add additional lines as necessary.  Each  line in this
+* letterfile cannot be longer than 126 characters. Each parameter setting
+* consists of a parameter name followed by a colon and the parameter value
+*  (for example: 'CT: 1')
+* Where instructions call for entering a range of values, enter single
+* or group values seperated by commas - ex. 1,10-20,30,33.
+*
+* PROGRAM PARAMETER SETTINGS START HERE ***************************************
+*------------------------------------------------------------------------------
+* INVALID HOME ACCOUNT TYPES
+*------------------------------------------------------------------------------
+*  Line must begin with "IHAT:"
+*
+*  Enter the account types which will exclude an account from being
+*  processed by this program.
+*
+*  If left blank, all open account types will be targeted
+*
+*  VALID VALUES: range of values from 0-9999
+*  DEFAULT: (left blank - account type not a factor)
+IHAT:
```

Comment:
I think it would be a good idea to also make an ineligible account Warning code Parameter, or an ineligible home account and an ineligible target account type Warning code list Parameters.

This way CU’s could prevent certain Members from Cross account access.

Do you think that makes sense to add that, Tom?

Reply:
Makes sense but we'll need to discuss this with Tiffany and Emily.  We could always add that functionality to v. 1.1.0 as well.

Reply:
Either one sounds good!